e2e/regression_settings/settings.no_max_basal.spec.js

```diff
@@ -0,0 +1,17 @@
+const { loop } = require('../../src/index');
+
+describe('Closed loop is not allowed when settings are not applied for max basal', () => {
+    it('launch loop', async () => {
+        await loop.Launch();
```

Comment:
[nit] If all of these functions being called (e.g. `loop.Launch()`, `loop.screens.settings.Open`) internally use `await`, is it possible and/or a good idea to get rid of the `await` at this level? Is it redundant?

Reply:
In looking through some of the code, I see multiple levels of `await` which I don't think are necessary. It would be good to come up with a common process on where `await` is called and where it is not.

Reply:
agreed as discussed they are required unfortunately